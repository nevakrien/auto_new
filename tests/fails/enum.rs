use new_macro::new;

#[derive(new)]
enum Dumby{
	A(u8),
	B(bool)
}