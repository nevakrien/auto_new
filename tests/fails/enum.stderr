error: only structs are allowed with #[derive(new)]
 --> tests/fails/enum.rs:4:6
  |
4 | enum Dumby{
  |      ^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fails/enum.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fails/enum.rs`
